studio/lib/helpers.ts

```diff
@@ -162,10 +162,11 @@ export const copyToClipboard = (str: string | Promise<string>, callback = () =>
   const focused = window.document.hasFocus()
   if (focused) {
     if (window.ClipboardItem) {
-      const text = new ClipboardItem({
-        'text/plain': Promise.resolve(str).then((text) => new Blob([text], { type: 'text/plain' })),
-      })
-      window.navigator?.clipboard?.write([text]).then(callback)
+      Promise.resolve(str)
```

Comment:
replace with `writeText`

Reply:
please use this as reference https://www.w3.org/TR/clipboard-apis/#dom-clipboard-writetext

Reply:
looks like writeText is already used below, so this is completely redundant. can be removed.